src/main/resources/data/computercraft/lua/rom/programs/edit.lua

```diff
@@ -47,6 +47,31 @@ else
     stringColour = colours.white
 end
 
+local runHeader = [[local current = term.current()
+local ok,err = load([===============]] .. [[==================[]]
+local runHandler = [[]===============]] .. [[==================], "@.temp", nil, _ENV)
```

Comment:
Instead of doing this horror story, could we do something like the following:

```lua
local runHandler = [[
local current = term.current()
local ok, err = load(%q, %q, nil, _ENV)
...
]]
```


And then:

```lua
local str = runHandler:format(table.concat(tLines, "\n"), "@" .. fs.getName(sPath))
```

Probably also worth dropping the `highlightColour`/`bgColour` - I'd just make those white and black respectively.

Reply:
I will admit i completely forgot that `%q` option exists in lua.